### PR TITLE
cdargs: Add package

### PR DESCRIPTION
**Pull request by nikil**
Maintainer: me
Compile tested: ar71xx, x86_64
Run tested: x86_64

Description: bookmarks and browsing for the cd command

Signed-off-by: Nikil Mehta nikil.mehta@gmail.com

### CODE REVIEW

File: utils/cdargs/Makefile [hunk 1]
```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (C) 2006-2016 OpenWrt.org
```
Comment (diizzyy) [inline utils/cdargs/Makefile]: Please don't use OpenWrt.org copyright unless you are under contract.
https://www.lede-project.org/faq.html#who_holds_the_copyright_of_lede_code

File: utils/cdargs/Makefile [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=cdargs
+PKG_VERSION:=1.35
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=http://www.skamphausen.de/downloads/cdargs/
```
Comment (diizzyy) [inline utils/cdargs/Makefile]: You can also add https://fossies.org/linux/privat/ as mirror

**Comment by diizzyy** [comment]
...also, why are you patching man pages that aren't installed?

**Comment by nikil** [comment]
Done - thanks for the comments.

File: utils/cdargs/Makefile [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  TITLE:=bookmarks and browsing for the cd command
+  URL:=http://www.skamphausen.de/cgi-bin/ska/CDargs
+  DEPENDS:=+libncursesw +libstdcpp
+endef
+
+define Package/cdargs/description
+	CDargs is a tool which enhances the navigation of the common unix file-system
+	inside the shell. It plugs into the shell built-in cd-command (via a shell
+	function or an  alias) and adds bookmarks and a browser to it.
+	.
```
Comment (diizzyy) [inline utils/cdargs/Makefile]: Typo?

**Comment by diizzyy** [comment]
@nikil 
If you still want this in, please fix your signed off line. I don't see any other issues execept the typo(?) mentioned above.
@hnyman @champtar Anything else to add?

**Comment by diizzyy** [comment]
@hnyman @thess 
Close due to submitter timeout

**Comment by neheb** [comment]
Closing due to request.

@nikil if you want this in, please create a new pull request.

**Pull request closed without merging**